projects/vdk-plugins/vdk-ingest-http/src/vdk/plugin/ingest_http/ingest_http_plugin.py

```diff
@@ -69,6 +69,13 @@ def vdk_configure(config_builder: ConfigurationBuilder) -> None:
         default_value=None,
         description="A string of comma-separated HTTP status codes that we should force a retry on.",
     )
+    config_builder.add(
+        key="INGEST_OVER_HTTP_ALLOW_NAN",
+        default_value=False,
+        description="If set to True, then NaN, Infinity, and -Infinity will be encoded as such."
+        "This behavior is not JSON specification compliant, but is consistent with most JavaScript based "
+        "encoders and decoders. Otherwise, it will be a ValueError to encode such floats.",
```

Comment:
What does the last sentence mean? Maybe it could be paraphrased?

Reply:
Oh, this is from simplejson encoder documentation on the parameter we pass, literally.
I just realised, maybe json.dumps() pydoc would be even better, updated the MR